### PR TITLE
feat: disable upload document animation for unverified users

### PR DESCRIPTION
**Pull request by dephraiim**
CleanShot 2023-12-09 at 22 01 48@2x

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```
Comment (Mythie) [inline .vscode/settings.json:4]: 👀

File: packages/ui/primitives/document-dropzone.tsx [hunk 2]
```diff
@@ -104,11 +104,12 @@ export const DocumentDropzone = ({
 
   return (
     <motion.div
-      className={cn('flex', className)}
+      className={cn('flex aria-disabled:cursor-none', className)}
```
Comment (Mythie) [inline packages/ui/primitives/document-dropzone.tsx:107]: `cursor-not-allowed` would be better I think, good use of aria-disabled though I was going to ask for it but forgot to

File: packages/ui/primitives/toast.tsx [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import * as React from 'react';
```
Comment (Mythie) [inline packages/ui/primitives/toast.tsx:1]: Old code 😇

**Comment by dguyen** [comment]
Can we add an actual message in the dropdown indicating why it's disabled?

File: .vscode/settings.json (hunk 1 above)
Comment (dephraiim) [reply .vscode/settings.json:4]: fixed

**Review by dguyen — APPROVED** [review]

**Pull request merged**